scripts/provision.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This file is here because upstream test scripts ostree-ng.sh will call it.
+#
+# The steps in this file are moved to rhel-edge-84-virt-test.sh, which is the main script to test RHEL 8.4
```

Comment:
This comment is not correct after your update.

Reply:
ok, will fix it.